src/synthcity/plugins/generic/plugin_marginal_distributions.py

```diff
@@ -35,7 +35,7 @@ class MarginalDistributionPlugin(Plugin):
 
     def __init__(self, **kwargs: Any) -> None:
         super().__init__(
-            sampling_strategy="marginal",
+            sampling_strategy="marginal", **kwargs
```

Comment:
On my end it shows that `black` reformats this line to

```python
super().__init__(sampling_strategy="marginal", **kwargs)
```

I am not sure why `black` isn't doing the same on MacOS, since the version is pinned in the `.pre-commit` config.